src/shared-components/accordion/index.tsx

```diff
@@ -80,7 +82,7 @@ export const Accordion = ({
             onClick(event);
           }
         }}
-        borderRadius={borderRadius}
+        borderRadius={borderRadiusValue}
```

Comment:
I think, wherever possible, it might be preferable to kick this `borderRadius` to `borderRadiusValue` logic into the `styled` component to hide it from the component logic markup. I know i've been guilty of this in the past with default colors, and we're already pulling in `theme` for the icon fill, but only after seeing more and more examples have I realized a preference against it 😅. I think we should save it for when we need to use it, like the Icon cases.